src/xdocs/google_style.xml

```diff
@@ -799,17 +799,21 @@
                 <td>
                   <span class="wrapper inline">
                     <img
-                        src="images/ok_blue.png"
+                        src="images/ok_green.png"
                         alt="" />
                   </span>
                   <a href="checks/sizes/linelength.html#LineLength">LineLength</a>
                   <br />
-                  We can detect URL with protocol type as http://, https:// etc.
                   <br />
                   <a href="https://webtoolkit.googleblog.com/2008/07/getting-to-really-know-gwt-part-1-jsni.html">
                     JSNI</a>
-                  could not be detected right now, but might be possible after
-                  comments and javadoc support appear in Checkstyle.
+                  is not supported by Checkstyle for this rule.
+                  After careful consideration and discussion,
+                  the Checkstyle maintainers have decided not to implement this
+                  feature in the foreseeable future. For more details regarding this decision,
+                  please refer to the discussion in the following issue:
+                  <a href="https://github.com/checkstyle/checkstyle/issues/14938">
+                    #14938</a>.
```

Comment:
Too much words.
Please do `JSNI is not supported, see reason at #14938`

Reply:
done